config/formats.js

```diff
@@ -660,6 +660,21 @@ let Formats = [
 		banlist: ['DUber', 'Gravity ++ Hypnosis', 'Gravity ++ Lovely Kiss', 'Gravity ++ Sing', 'Gravity ++ Sleep Powder'],
 		requireGalar: true,
 	},
+	{
+		name: "[Gen 8] LC (Beta) ",
+		threads: [`&bullet; <a href="https://www.smogon.com/forums/threads/metagame-discussion-thread.3656348/">Metagame Discussion</a>`,
+			
+		],
+
+		mod: 'gen8',
+		maxLevel: 5,
+		ruleset: ['Obtainable', 'Standard' , 'Team Preview', 'Little Cup'],
+		banlist: [
+			'Sneasel', 'Swirlix',
+	 'Baton Pass', 'Dragon Rage', 'Sonic Boom',
+		], 
```

Comment:
This should all be one line. Alsom Dragon Rage and Sonic Boom aren't in Gen 8 so there's no need to ban them.